tools/c7n_gcp/c7n_gcp/mu.py

```diff
@@ -720,7 +720,7 @@ def get_subscription(self, func):
         log_name = "{}/{}/logs/cloudaudit.googleapis.com%2Factivity".format(
             self.data.get('scope', 'projects'),
             self.session.get_default_project())
-        log_filter = 'logName = "%s"' % log_name
+        log_filter = '(logName = "%s" OR resource.type = "audited_resource")' % log_name
```

Comment:
can you elaborate why this is needed here. effecting the generic query for all resources on the basis of a specific need around a single one, is suspect and has wider implications.